### ISSUE TITLE
failed to load source : axios error  

### ISSUE THREAD

**Issue by brygulley25** [Elliott-Chong/chatpdf-yt#43]
○ Compiling /sign-in/[[...sign-in]] ...
 ✓ Compiled /sign-in/[[...sign-in]] in 1214ms (1764 modules)
 ○ Compiling /api/create-chat ...
 ✓ Compiled /api/create-chat in 4.2s (1011 modules)
uploads/1709102023870Educational-E-book Doc.pdf Educational E-book Doc.pdf
downloading s3 into file system
Error: Resolved credential object is not valid is the terminal error i get, after i upgraded to standard pinecone db and all credentials are correct with no typos, and i also have these errors in the console looking like its having a hard time posting the vectors into pinecone and im not sure why. followed along with the video and been doing research and found nothing. anything helps 
![Screenshot 2024-02-28 013911](https://github.com/Elliott-Chong/chatpdf-yt/assets/121139682/bc761d9d-4dd8-4a46-b14b-b3cb185979c3)

**Comment by kapcodeO** [comment]
Got the same error ..

**Comment by jma49** [comment]
well...same error..
<img width="573" alt="image" src="https://github.com/Elliott-Chong/chatpdf-yt/assets/83278146/a9889d1a-d392-4d7e-9884-91d0a4eb4166">

**Comment by ThisIsMyName678** [comment]
@brygulley25 did you solved it?

**Comment by brygulley25** [comment]
Haven't solved yet

**Comment by Sudipta013** [comment]
yes i am getting the same error. Have you guys solved it? or found any solutions?

**Comment by rkf2778** [comment]
Try adding console log or try catch error to yout `s3.ts` file
